cross-chain/bob/contracts/interfaces/IOptimismMintableERC20.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC165} from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+
+/// @title IOptimismMintableERC20
+/// @notice This interface is available on the OptimismMintableERC20 contract.
+///         We declare it as a separate interface so that it can be used in
+///         custom implementations of OptimismMintableERC20.
+interface IOptimismMintableERC20 is IERC165 {
```

Comment:
catching up with the canonical version you mention. 

- does the canonical convergence is just partial here?
- do we need to approximate as much as possible the version `solidity/contracts/cross-chain/wormhole/L2TBTC.sol` ??
I understand that maybe we cannot converge 100% on this case.

Reply:
We are mostly porting the canonical methods, modifiers and variables to the existing BOB TBTC contract. 
We are not entirely removing the previous existing functionalities, but rather upgrading/adapting them due to the bridge transitioning aspect. So it should continue to accept the BOB Standard Bridge in the legacy format, while 100% compatible with the canonical L2TBTC token.